student-notebooks/02.07-RosettaScripts-in-PyRosetta.ipynb

```diff
@@ -191,9 +191,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "\n",
+    "parser = RosettaScriptsParser()\n",
+    "protocol = parser.generate_mover(\"inputs/min_L1.xml\")\n",
+    "\n",
     "if not os.getenv(\"DEBUG\"):\n",
-    "    parser = RosettaScriptsParser()\n",
```

Comment:
@kathyle9 @lyskov @jadolfbr can you guys check this edit? not sure this is related to adding in Ch 9